app/models/Course.php

```diff
@@ -443,8 +443,8 @@ public static function score_distribution($courseid) {
                 SELECT hole.hole_num,
                 CASE WHEN score.stroke = 1 THEN 1 END AS hole_in_one,
                 CASE WHEN hole.par - score.stroke - score.ob = 2 AND score.stroke != 1 THEN 1 END AS eagle,
-                CASE WHEN hole.par - score.stroke - score.ob = 1 THEN 1 END AS birdie,
-                CASE WHEN hole.par - score.stroke - score.ob = 0 THEN 1 END AS par,
+                CASE WHEN hole.par - score.stroke - score.ob = 1 AND score.stroke != 1 THEN 1 END AS birdie,
+                CASE WHEN hole.par - score.stroke - score.ob = 0 AND score.stroke != 1 THEN 1 END AS par,
```

Comment:
The first case should already catch it when stroke = 1?

Reply:
It should'nt catch it because each of them are separate cases. One case doesn't affect to other cases.

Reply:
Oh I see

Reply:
Then rather remove this check for par and add it to the first case as I described in the other comment?